WeakAuras/GenericTrigger.lua

```diff
@@ -2899,32 +2939,17 @@ function GenericTrigger.GetTriggerConditions(data, triggernum)
       end
 
       if (timedDuration) then
-        result["expirationTime"] = {
-          display = L["Remaining Duration"],
-          type = "timer",
-        }
-        result["duration"] = {
-          display = L["Total Duration"],
-          type = "number",
-        }
+        result.expirationTime = CopyTable(commonConditions.expirationTime);
```

Comment:
Why create a copy of the table instead of just using a reference? (similarly on other calls to CopyTable)

Reply:
No particular reason. In this case it should not be needed as those tables are not meant to be changed.